packages/react-data-grid/src/Cell/CellContent.tsx

```diff
@@ -1,10 +1,11 @@
-import React from 'react';
+import React, { createElement, cloneElement } from 'react';
```

Comment:
I am wondering if this layer is needed as well. However for this particular PR, I would prefer less removal, so removing `CellValue` should be enough.